app/controllers/cars_controller.rb

```diff
@@ -1,13 +1,19 @@
 class CarsController < ApplicationController
     # skip_before_action :verify_authenticity_token
+    # CODE_COMMENT: I see that most of the controller / view code you have written is in cars any reason for this? 
+    # Probabaly you could have also used dashboard controller well?
+    # CODE_COMMENT: Why is the above line there?
     before_action :authenticate_user!, except: [:index, :show, :allcars]
     before_action :correct_user, only: [:edit, :update, :destroy, :dashboard]
 
+    # CODE_COMMENT: Are we sure we want all the cars for the next 3 statements? Isnt that an overkil? We should do some sort of user related car fetching 
+    # or something on those lines correct?
```

Comment:
def allcars: this function needs to get all cars and display all cars.
def index: this function needs only cars that belong to certain users.